pkgs/top-level/python-packages.nix

```diff
@@ -9148,6 +9148,24 @@ in {
     };
   };
 
+  pysrt = buildPythonApplication rec {
```

Comment:
Please read: https://github.com/nixos-users/wiki/wiki/Python-libraries#contribution-guidelines
regarding adding packages directly `pkgs/top-level/python-packages.nix`. Also `buildPythonPackage` should be used here instead of `buildPythonApplication`.